ed25519-bip32/ed25519_bip32.h

```diff
@@ -1,28 +1,21 @@
 /* Warning, this file is autogenerated by cbindgen. Don't modify this manually. */
```

Comment:
Is this header file required to be written by hand?

I think https://github.com/spacemeshos/post-rs generates the C headers when it builds the library, maybe @poszu can help here?

Reply:
I don't understand the question. It's not written by hand. There's a makefile recipe for this.

Reply:
Ah my bad. It seems like https://github.com/spacemeshos/post-rs also generates the headers using a `build.rs` file, but doesn't commit them to the repository. I misunderstood the files to be manually edited 😞

Reply:
no strong opinion. personally I kind of like manually regenerating it, and being alerted when it changes.